docs/onboarding/8_16/operator/onprem_kube_stack_values.yaml

```diff
@@ -0,0 +1,407 @@
+                  enabled: true
+                process.open_file_descriptors:
+                  enabled: true
+                process.memory.utilization:
+                  enabled: true
+                process.disk.operations:
+                  enabled: true
+            network: {}
+            processes: {}
+            load: {}
+            disk: {}
+            filesystem:
+              exclude_mount_points:
+                mount_points:
+                  - /dev/*
+                  - /proc/*
+                  - /sys/*
+                  - /run/k3s/containerd/*
+                  - /var/lib/docker/*
+                  - /var/lib/kubelet/*
+                  - /snap/*
+                match_type: regexp
+              exclude_fs_types:
+                fs_types:
+                  - autofs
+                  - binfmt_misc
+                  - bpf
+                  - cgroup2
+                  - configfs
+                  - debugfs
+                  - devpts
+                  - devtmpfs
+                  - fusectl
+                  - hugetlbfs
+                  - iso9660
+                  - mqueue
+                  - nsfs
+                  - overlay
+                  - proc
+                  - procfs
+                  - pstore
+                  - rpc_pipefs
+                  - securityfs
+                  - selinuxfs
+                  - squashfs
+                  - sysfs
+                  - tracefs
+                match_type: strict
+        kubeletstats:
+          auth_type: serviceAccount
+          collection_interval: 20s
+          endpoint: ${env:K8S_NODE_NAME}:10250
+          node: '${env:K8S_NODE_NAME}'
+          # Required to work for all CSPs without an issue
+          insecure_skip_verify: true
+          k8s_api_config:
+            auth_type: serviceAccount
+          metrics:
+            k8s.pod.memory.node.utilization:
+              enabled: true
+            k8s.pod.cpu.node.utilization:
+              enabled: true
+            k8s.container.cpu_limit_utilization:
+              enabled: true
+            k8s.pod.cpu_limit_utilization:
+              enabled: true
+            k8s.container.cpu_request_utilization:
+              enabled: true
+            k8s.container.memory_limit_utilization:
+              enabled: true
+            k8s.pod.memory_limit_utilization:
+              enabled: true
+            k8s.container.memory_request_utilization:
+              enabled: true
+            k8s.node.uptime:
+              enabled: true
+            k8s.node.cpu.usage:
+              enabled: true
+            k8s.pod.cpu.usage:
+              enabled: true
+          extra_metadata_labels:
+            - container.id
+      service:
+        pipelines:
+          logs/node:
+            receivers: [filelog]
+            processors: [batch, k8sattributes, resourcedetection/system, resourcedetection/eks, resourcedetection/gcp, resource/k8s, resource/cloud, attributes/k8s_logs_dataset]
```

Comment:
nit: maybe consider defining the items of these arrays as one per line? This will make referencing and commenting out/in of specific items easier.

Reply:
@ChrsMark fair enough, changed in https://github.com/elastic/opentelemetry/pull/11/commits/253222c8cfb709824dc45fd6dc746d7a07783d77

Reply:
@mlunadia Definitely, I will do that once all the required checkpoints are done.